app/controllers/catalog_controller.rb

```diff
@@ -215,16 +215,25 @@ def lookup_manifest
     end
   end
 
+  # This endpoint redirects to download an original pdf if present,
+  # or redirects to the resource pdf endpoint if one needs to be generated
   def pdf
     _, @document = fetch(params[:solr_document_id])
 
     source_pdf = Wayfinder.for(resource).source_pdf
-    if source_pdf && can?(:download, source_pdf)
-      redirect_to "#{request.base_url}#{helpers.fileset_download_path(source_pdf)}"
+    if source_pdf
```

Comment:
Does the removal of the download permission check here change anything? I'm trying to think if there were cases where I could PDF but not download, is that a thing?

Reply:
Where you could download the pdf but not the original files? I'm not aware of any and it seems unlikely. but not outside the realm of possibility. I'm not really sure how to figure this out.

the downloads controller does authorize before allowing the download, which is why it seemed like removing this was okay.

https://github.com/pulibrary/figgy/blob/b03688010aab8a803c32750404bf8f8e05a9251c/app/controllers/downloads_controller.rb#L91-L94

Definitely appreciate confirmation on whether this seems right.